packages/schematics/angular/ng-new/schema.json

```diff
@@ -130,10 +130,10 @@
         "type": "list",
         "items": [
           { "value": "css",  "label": "CSS" },
-          { "value": "scss", "label": "SCSS   [ http://sass-lang.com/documentation/file.SASS_REFERENCE.html#syntax ]" },
-          { "value": "sass", "label": "Sass   [ http://sass-lang.com/documentation/file.INDENTED_SYNTAX.html       ]" },
-          { "value": "less", "label": "Less   [ http://lesscss.org                                                 ]" },
-          { "value": "styl", "label": "Stylus [ http://stylus-lang.com                                             ] " }
+          { "value": "scss", "label": "SCSS   [ https://sass-lang.com/documentation/syntax#scss                ]" },
+          { "value": "sass", "label": "Sass   [ https://sass-lang.com/documentation/syntax#the-indented-syntax ]" },
+          { "value": "less", "label": "Less   [ http://lesscss.org                                             ]" },
+          { "value": "styl", "label": "Stylus [ http://stylus-lang.com                                         ] " }
```

Comment:
Small nit, can you please remove the space after the closing square bracket here?

Also, please rebase to make this green thanks.